checkout-service/client/src/components/Checkout.jsx

```diff
@@ -1,119 +1,70 @@
-  font-size: 18px;
-  line-height: 24px;
-  margin-bottom: 10px;
-`;
-
-const smallerPadding = {
-  lineHeight: "20px"
-};
-
-const lineStyle = {
-  borderTop: "1px solid #D5D9D9"
-}
-
-
-const PriceDeliveryAndStock = (props) => (
-  <div>
-    <div style={{marginBottom: "10px"}}>
-      <Price>${props.price}</Price>
-      <img style={{width: "50px", height: "auto"}} src='https://m.media-amazon.com/images/G/01/AmazonServices/Site/US/Product/FBA/small-and-light-prime-logo._V509606070_.png'></img><GreyInlineText> FREE One-Day</GreyInlineText>
-      <div>
-        <InlineText>& </InlineText><BlueInlineText>FREE Returns <sup>&#8744;</sup></BlueInlineText>
-      </div>
-    </div>
-    <div style={{marginBottom: "20px"}}>
-      <Text style={smallerPadding}>FREE delivery: <strong>Tomorrow</strong></Text>
-      <GreyText style={smallerPadding}>Order within 'hours remaining'</GreyText>
-      <BlueText style={smallerPadding}>Details</BlueText>
-    </div>
-    <Stock style={props.inventory ? {color: "#007600"} : {color: "#B12704"}}>{props.inventory ? "In Stock." : "Currently unavailble."}</Stock>
-    {!props.inventory ? <Text style={{lineHeight: "20px"}}>We don't know when this item will be back in stock.</Text> : null}
-  </div>
-);
-
-const QuantityDropDown = (props) => (
-  <div>
-    <h2>QuantityDropDown</h2>
-  </div>
-);
-
-const AddToCartButton = (props) => (
-  <div>
-    <h2>AddToCartButton</h2>
-  </div>
-);
-
-const BuyNowButton = (props) => (
-  <div>
-    <h2>BuyNowButton</h2>
-  </div>
-);
-
-const SecureTransactionAndSellerDetails = (props) => (
-  <div>
-  <h2>SecureTransaction...</h2>
-</div>
-)
-
-const AddToListDropdown = (props) => (
-  <div style={lineStyle}>
-    <h2>AddToListDropdown</h2>
-  </div>
-);
-
-
-export {
+import {
   PriceDeliveryAndStock,
   QuantityDropDown,
   AddToCartButton,
   BuyNowButton,
-  SecureTransactionAndSellerDetails,
   AddToListDropdown,
-};
\ No newline at end of file
+  SecureTransactionAndSellerDetails
+} from './CheckoutDetails.jsx';
+
+
+const CheckoutWrapper = styled.div`
+  width: 242px;
+  max-width: 242px;
+  border: 1px solid #D5D9D9;
+  border-radius: 8px;
+  padding: 14px 18px;
+`;
+
+export default class Checkout extends React.Component {
+  constructor (props) {
+    super (props);
+    this.state = {
+      productId: 1001,
+      price: null,
+      inventory: null
+    };
+  }
+
+
+  componentDidMount() {
+    fetch(`http://localhost:4003/priceandinventory/id/${this.state.productId}`)
+    .then((res) => res.json())
+    .then((res) => {
+      // console.log('price: ', res[0].price, 'inventory: ', res[0].inventory);
+      let productPrice = res[0].price;
+      let productInventory = res[0].inventory;
+      this.setState({
+        price: productPrice,
+        inventory: productInventory
+      });
+    })
+  }
+
+
+  render () {
+
+    let propsToPass = { price: this.state.price, inventory: this.state.inventory }
+
+    return (
```

Comment:
I really like how this is laid out. It is easy for me to follow and visual the layout of your components. You do a good job naming your variables.